.bash_nav

```diff
@@ -7,3 +7,31 @@ alias desk='cd /mnt/c/Users/gsera/Desktop'
 # source bash
 
 alias srcbash='source ~/.bashrc'
+
+# ---This alias effectively creates a bookmarking app---
+# it adds global symlink shortcuts for use with the 'cds' command
+# ex: 'cds studycloud' will run 'cd -P ~/.short/studycloud'
+# to add a shortcut: go to the target directory and run 'cds + <shortcut_name>' or 'cds + .' to use the current directory name
+# to remove a shortcut: run 'cds - <shortcut_name>' (you can also tack on multiple shortcut names separated by spaces)
+# to list your shortcuts: run 'cds .'
+cds() {
+    if [ ! -d ~/.short ]; then
+        mkdir ~/.short
+    fi
+    if [ $# -eq 1 ] && [ $1 = "." ]; then
+        ls ~/.short/
+    elif [ $# -eq 2 ] && [ $1 = "+" ]; then
+        if [ $2 != "-" ]; then
+            ln -s "$PWD" "$HOME/.short/$2"
```

Comment:
Create a symlink to the current directory and store it in the `.short` directory. Name it the specified name.
If the second argument is `.`, the name will be the name of the current directory.